client/flutter/lib/l10n/intl_en_US.arb

```diff
@@ -3,16 +3,31 @@
   "commonWorldHealthOrganization": "World Health Organization",
   "commonWorldHealthOrganizationCoronavirusApp": "COVID-19 App",
   "commonWhoAppShareIconButtonDescription": "Check out the official COVID-19 Guide App https://whocoronavirus.org/app",
-  "commonWorldHealthOrganizationCoronavirusAppVersion": "Version 0.1 (12412)\n© 2020 WHO",
+  "commonWorldHealthOrganizationCoronavirusAppVersion": "Version {version} ({buildNumber})",
+  "commonWorldHealthOrganizationCoronavirusCopyright": "© {year} WHO",
   "commonPermissionRequestPageButtonSkip": "Skip",
+  "commonDialogButtonOk": "OK",
+  "commonContentLoadingDialogUpdateRequiredTitle": "App Update Required",
+  "commonContentLoadingDialogUpdateRequiredBodyText": "Please update to the latest version of the app in order to receive the latest information and updates.",
+  "@section": "Legal Landing Page",
+  "legalLandingPageTitle": "Official WHO COVID-19 Information App",
+  "legalLandingPageButtonGetStarted": "Get Started",
+  "legalLandingPageButtonAgree": "By proceeding, you agree to our",
+  "LegalLandingPageTermsOfServiceLinkText": "Terms of Service",
+  "legalLandingPageTermsOfServiceLinkUrl": "https://whocoronavirus.org/terms",
+  "legalLandingPageAnd": "and",
```

Comment:
must preserve the spaces here " and "

Reply:
Ok, I can do but that's bad practice to rely on strings content to format the app. the right thing to do is to add padding around it, handling the style properly.